src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1></h1>
+    <div class="searchbar">
+
+      <form action="#"
+        method="post"
+        name="form"
+        data-qa="big"
+        class="searchbar__big">
+
+
+        <div class="searchbar__big--input">
+          <input type="search"
+          name="big-form"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="searchbar__big--input--input">
```

Comment:
```suggestion
          <input 
            type="search"
            name="big-form"
            data-qa="keypress"
            placeholder="Try \"Los Angeles\""
            class="searchbar__input searchbar__input--big"
          >
```
and make second in this way